src/BlazorAdmin/Services/CatalogItemServices/CatalogItem.cs

```diff
@@ -17,11 +17,12 @@ public class CatalogItem
         [Required(ErrorMessage = "The Name field is required")]
         public string Name { get; set; }
 
+        [Required(ErrorMessage = "The Description field is required")]
         public string Description { get; set; }
 
         // decimal(18,2)
         [RegularExpression(@"^\d+(\.\d{0,2})*$", ErrorMessage = "The field Price must be a positive number with maximum two decimals.")]
-        [Range(0, 9999999999999999.99)]
+        [Range(0.01, 9999999999999999.99)]
```

Comment:
We can probably limit price to something reasonable like 1000.

Reply:
Done